app/containers/Toast.tsx

```diff
@@ -25,50 +25,32 @@ interface IToastProps {
 	theme?: TSupportedThemes;
 }
 
-class Toast extends React.Component<IToastProps, any> {
-	private listener?: Function;
-
-	private toast: EasyToast | null | undefined;
-
-	componentDidMount() {
-		this.listener = EventEmitter.addEventListener(LISTENER, this.showToast);
-	}
-
-	shouldComponentUpdate(nextProps: any) {
-		const { theme } = this.props;
-		if (nextProps.theme !== theme) {
-			return true;
-		}
-		return false;
-	}
-
-	componentWillUnmount() {
-		if (this.listener) {
-			EventEmitter.removeListener(LISTENER, this.listener);
-		}
-	}
-
-	getToastRef = (toast: EasyToast | null) => (this.toast = toast);
-
-	showToast = ({ message }: { message: string }) => {
-		if (this.toast && this.toast.show) {
-			this.toast.show(message, 1000);
+const Toast = ({ theme }: IToastProps) => {
```

Comment:
If you remove the `withTheme` HOC and use the `useTheme` hook you won't need this, you could use `colors`
```ts
const { colors } = useTheme();
```